src/pages/RFOX/components/Stake/StakeConfirm.tsx

```diff
@@ -62,6 +66,19 @@ export const StakeConfirm: React.FC<StakeConfirmProps & StakeRouteProps> = ({
   const stakingAsset = useAppSelector(state =>
     selectAssetById(state, confirmedQuote.stakingAssetId),
   )
+  const stakingAssetFeeAsset = useAppSelector(state =>
+    selectFeeAssetByChainId(state, fromAssetId(confirmedQuote.stakingAssetId).chainId),
```

Comment:
suggestion:

```suggestion
    selectFeeAssetByChainId(state, stakingAsset.chainId),
```

Reply:
would be noicer but then we'd have to opt-chain to empty string :(
<img width="915" alt="image" src="https://github.com/shapeshift/web/assets/17035424/c2e27653-1996-46a4-9eb1-04b6ddefee04">

Reply:
these selectors man..